config/Zed/oms/DummyPayment01.xml

```diff
@@ -29,6 +29,7 @@
                 <flag>cancellable</flag>
             </state>
             <state name="exported" reserved="true" display="oms.state.exported"/>
+            <state name="product review requested" reserved="true" display="oms.state.paid"/>
```

Comment:
if the transition is `paid` -> `product review requested`, please put it after `<state name="paid" ...` to improve visibility.

Reply:
fixed